.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,8 +5,9 @@ Resolves
 * CI/run-audit-deps (1) - check for known npm/cargo vulnerabilities (audit_deps)
 * CI/run-network-audit (1) - run network-audit
 * CI/run-upstream-tests - run Chromium unit and browser tests on Linux and Windows (otherwise only on Linux)
+* CI/run-linux-arm64, CI/run-windows-arm64, CI/run-windows-a86 - run builds that would otherwise be skipped
```

Comment:
`CI/run-windows-a86` has a typo. Also, are we going to skip Windows x86 from now on?

Reply:
We have a big user base on x86 and it's probably good to test on 2 Windows platforms rather than 1 since most of our desktop users are Windows.